### DISCUSSION TITLE
Recipient folder in my phone not taken into account.

### DISCUSSION THREAD

**Discussion by Franck-DUBUS** [localsend/localsend#1695] [Discussion (English, International)]
Friday 08/23/24 (Reposted today)
Good evening ;
Last 5 and 16/08, I "posted" the question below, although "polite" but remained a dead letter! !  I didn't expect such a rejection!! I put it back for any useful purpose ::

Good morning ;
I installed LocalSend, it works well, except that when I transfer one (of) files from my computer with Windows 10 Pro, (updated) to my One Plus Nord smartphone with Android 12, it does not fit in the named folder "LocalSend download" that I created in "OnePlus Nord\Internal storage", but it always happens in the "Pictures" folder I don't know how to resolve this problem, thank you for your help.
To make it a little simpler, how do I send a file/folder to the folder I created "LocalSend Download" in the storage of my OnePlus Nord smartphone
Sincerely.

**Comment by Tienisto** [comment]
You can disable "Save media to gallery / photos" option in the app settings.

**Reply by Franck-DUBUS** [reply]
Good morning ;
I want to, but sorry, because I can't find "Save media to gallery/photos" in the settings.
Another thing, at the beginning, it worked, apart from this problem of destination folder, but now, no way to make the phone and the PC communicate with each other!! I don't know what to do anymore, you have to send me a precise tutorial, because I can't cope, I have the portable version, I tried to take the version to install, it's worse, with it , I'm "Offline" in the firewall, it's still good, Wifi too, I'm tired of struggling with that, unless I get help from you, I'm going to give up, it's too much "puzzle"
Another thing, during a version update, how do you keep your settings, whether with the portable or installed version?
To read you, thank you.

**Comment by Franck-DUBUS** [comment]
Good morning ;
I want to, but sorry, because I can't find "Save media to gallery/photos"
in the settings.
Another thing, at the beginning, it worked, apart from this problem of
destination folder, but now, no way to make the phone and the PC
communicate with each other!! I don't know what to do anymore, you have to
send me a precise tutorial, because I can't cope, I have the portable
version, I tried to take the version to install, it's worse, with it , I'm
"Offline" in the firewall, it's still good, Wifi too, I'm tired of
struggling with that, unless I get help from you, I'm going to give up,
it's too much "puzzle"
Another thing, during a version update, how do you keep your settings,
whether with the portable or installed version?
To read you, thank you.

Le ven. 23 août 2024 à 22:53, Tien Do Nam ***@***.***> a
écrit :

**Reply by Tienisto** [reply]
You need to backup the `settings.json` file to keep the settings across different versions in portable mode.
The "Save media to gallery" option is below "Save to folder" setting (see https://github.com/localsend/localsend/issues/1701 for screenshot).

Regarding discovery issues, please lookup on Github, there are plenty other users with different methods to solve the issue. I don't have time for private support.
